tests/main/refresh-all/task.yaml

```diff
@@ -0,0 +1,36 @@
+summary: Check that more than one snap is refreshed.
+
+details: |
+    We use only the fake store for this test because we currently
+    have only one controlled snap in the remote stores, when we will
+    have more we can update the test to use them
+
+environment:
+    BLOB_DIR: $(mktemp -d)
+
+prepare: |
+    . $TESTSLIB/store.sh
+
+    echo "Given two snaps are installed"
+    for snap in hello-world xkcd-webserver; do
```

Comment:
Not for this branch, but we need to go ahead with that plan we discussed, only using snaps named `test-snapd-*` from the store and reducing them to a minimum set that make a difference for testing.